tester/tester.go

```diff
@@ -193,9 +194,18 @@ func validateHTTPCode(contract Contract, resp *http.Response) error {
 
 func validateResponseBody(contract Contract, body []byte) error {
 	if contract.ExpectedResponseBody != "" {
-		if !strings.Contains(string(body), contract.ExpectedResponseBody) {
-			return fmt.Errorf("expected response not found in the body")
+		if !strings.HasPrefix(contract.ExpectedResponseBody, "r/") {
+			if !strings.Contains(string(body), contract.ExpectedResponseBody) {
+				return fmt.Errorf("expected response not found in the body")
+			}
+		} else {
+			expectedRegexp := contract.ExpectedResponseBody[2:]
+			re := regexp.MustCompile(expectedRegexp)
+			if !re.MatchString(string(body)) {
```

Comment:
would use regexp.Match and avoid the `string(body)` conversion

Reply:
Done